services/question-service/src/tests/e2e.test.ts

```diff
@@ -0,0 +1,366 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import assert from "assert";
+import { getToken } from "next-auth/jwt";
+import request from "supertest";
+
+import { Category, Difficulty } from "@prisma/client";
+
+import { loadEnvConfig } from "../commons/utils/env-config";
+import { getPrismaClient } from "../data-access/prisma-client-factory";
+import { createWebApplication } from "../entry-points/api/server";
+
+import resetDb from "./helper/resetDb";
+
+const app = createWebApplication();
+
+const mockApp = request(app);
+
+const prisma = getPrismaClient();
+
+loadEnvConfig();
+
+jest.mock("next-auth/jwt");
+
+beforeEach(async () => {
+  await resetDb();
+  jest.clearAllMocks();
+  (getToken as jest.Mock).mockResolvedValue(null);
```

Comment:
What does this do?

Reply:
just sets a default token result if no token provided ah